joblib/memory.py

```diff
@@ -344,6 +344,10 @@ class NotMemorizedFunc(object):
     # Should be a light as possible (for speed)
     def __init__(self, func):
         self.func = func
+        try:
+            functools.update_wrapper(self, func)
+        except:
+            "Objects like ufunc don't like that"
```

Comment:
Could you put the wrapping code in an helper and use it for both `NotMemorizedFunc` and `MemorizedFunc`?

Also, could you also set the doc string of the function, as it is done for `MemorizedFunc`?